scripts/setup-github-token.ps1

```
@@ -0,0 +1,122 @@
+# Guides the user through setting up a GitHub Personal Access Token
+# Stores it as a user-level environment variable
+
+param(
+    [switch]$Test,
+    [switch]$DryRun,
+    [switch]$Help
+)
+
+function Write-ColorOutput {
+    param([string]$Message, [string]$Color = "White")
+    Write-Host $Message -ForegroundColor $Color
```

PSScriptAnalyzer (1.24.0)

[warning] File 'setup-github-token.ps1' uses Write-Host. Avoid using Write-Host because it might not work in all hosts, does not work when there is no host, and (prior to PS 5.0) cannot be suppressed, captured, or redirected. Instead, use Write-Output, Write-Verbose, or Write-Information.

(PSAvoidUsingWriteHost)